src/Nest/Mapping/Types/Core/String/StringProperty.cs

```diff
@@ -36,9 +36,13 @@ public interface IStringProperty : IProperty
 		[JsonProperty("ignore_above")]
 		int? IgnoreAbove { get; set; }
 
-		[JsonProperty("position_offset_gap")]
+		[JsonIgnore]
+		[Obsolete("Scheduled to be removed in 5.0. Use PositionIncrementGap instead.")]
 		int? PositionOffsetGap { get; set; }
```

Comment:
Lets make these set PositionIncrementGap too so we don't break for folks ignoring obsolete messages